package.json

```diff
@@ -44,11 +44,14 @@
     "@angular/platform-browser": "4.4.3",
     "@angular/platform-browser-dynamic": "4.4.3",
     "@angular/router": "4.4.3",
+    "angular-gridster2": "^3.15.0",
+    "bootstrap": "^3.3.7",
```

Comment:
See my comment for src/index.html in regards to the bootstrap and the jQuery dependencies here